README.md

```diff
@@ -51,53 +51,55 @@ $ yarn install
 # Run website
 $ yarn dev
 ```
-This should launch the website on `localhost:3000`.
+
+<strong> This should launch the website on `localhost:3000`. <strong>
 
 We have used proxy so all the requests made to backend from frontend will be served at
 `localhost:5000` at which our backend is running.
 
-### <strong>Note 👀:</strong>
+
+<h2>Note <a><img src="https://github.com/Samridhi-98/MERN-Gurujii-dev/blob/master/readme_image/alarm.svg" width="3%"></a> </h2>
+<p>
+ 
 - This project uses yarn to install the dependencies. Don't use npm install.
 - For security reasons we have made the MongoDb URI secure, by not adding a hardcoded username and password. For development purpose, you can install, MongoDB locally or use the  online version.
 - Comment line number 6,7,22 in backend/server.js.
 - If you have use the  online version add your url in line number 22.
 - Comment line number 4 in backend/routes/users.js.
 - Add your auth and pass in backend/routes/users.js line number 94 and 95.
-<br>
-<br>
-<br>
 
-### <strong>Want to test your changes on mobile phone?</strong>
+</p>
+
+<h2>Contribution <a><img src="https://github.com/Samridhi-98/MERN-Gurujii-dev/blob/master/readme_image/support(1).svg" width="3%"></a> </h2>
+<p>
+ Please check this link <a href="https://github.com/Namanl2001/MERN-Gurujii-dev/blob/master/contributing.md">contribution.md </a> before start contributing and making pull request.
```

Comment:
`Contributions are very welcome! See our CONTRIBUTING.md for more information.`

Reply:
Okay, will do it.